appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/lpa-questionnaire.controller.js

```diff
@@ -375,19 +387,36 @@ export const getAddDocumentsCheckAndConfirm = async (request, response) => {
 
 /** @type {import('@pins/express').RequestHandler<Response>} */
 export const postAddDocumentsCheckAndConfirm = async (request, response) => {
-	const { currentAppeal } = request;
+	const { currentAppeal, currentFolder } = request;
 
-	if (!currentAppeal) {
+	if (!currentAppeal || !currentFolder) {
 		return response.status(404).render('app/404');
 	}
 
 	try {
+		const documentType = currentFolder.path.split('/')[1];
+		let notificationBannerText = '';
+
+		switch (documentType) {
+			case `${APPEAL_DOCUMENT_TYPE.OTHER_RELEVANT_POLICIES}`:
+				notificationBannerText = 'Other relevant policies added';
```

Comment:
On second thought I'm going to push back to design/content on this - there doesn't seem to be any reason for this copy to differ from other folder rows